### PR TITLE
Added mapbox_gl widget with updated build.gradle

### PR DESCRIPTION
**Pull request by mysticetii**
It is for https://github.com/Geovation/flutter_sandbox/issues/5.
- Updated minSdkVersion to 20 as mapbox widget requires it.
- Added full page mapbox widget with full-screen map
- Access token is required and insert your token from mapbox into lib>mapbox>constants_mapbox.dart file in the ACCESS_TOKEN variable.

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -1,4 +1,7 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_sandbox/mapbox/mapbox_page.dart';
+
+import 'mapbox/mapbox_page.dart';
```
Comment (joykuotw) [inline lib/main.dart:4]: Line 2 and 4 are doing the same import. Delete one of them.

**Review by joykuotw — CHANGES_REQUESTED** [review]

**Comment by joykuotw** [comment]
What version of iOS simulator and Android emulator do you use to run the app?

File: lib/main.dart (hunk 1 above)
Comment (mysticetii) [reply lib/main.dart:3]: Done. I have just pushed another commit.

**Comment by mysticetii** [comment]
I am developing this project using Android studio.
I do not have access to macOS therefore I am not using iOS simulator. For Android I am using emulator version: 30.4.5-7140946 and android version: 10.0 (Q) - API 29.

**Review by joykuotw — APPROVED** [review]

**Pull request merged**